.github/workflows/build-apps.yml

```diff
@@ -202,21 +207,21 @@ jobs:
           smksp_cert_sync.exe
           smctl windows certsync
         # This last line `smctl windows certsync` was added after windows codesign failures started happening
-        # with nightly-v25.4.10. It looks like `smksp_cert_sync.exe` used to do the sync to the local cert store,
+        # with staging-v25.4.10. It looks like `smksp_cert_sync.exe` used to do the sync to the local cert store,
```

Comment:
I think you want to revert this one line or just delete the whole comment block.